src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -310,6 +310,7 @@ WebGLRenderer.prototype.setObjectRenderer = function (objectRenderer)
     }
 
     this.currentRenderer.stop();
+    this.currentRenderer.destroy();
```

Comment:
I don't think this is desired, this would destroy the object renderers everytime you switched to a different renderer. For example, if you render a sprite then a graphics object when you render the graphics object the sprite renderer will be destroyed.

The object renderers are registered as plugins of the webgl renderer and should be destroyed in the destroy method when all other plugins are destroyed.